package.json

```diff
@@ -27,11 +27,15 @@
     "@babel/preset-react": "^7.14.5",
     "@babel/preset-typescript": "^7.14.5",
     "@babel/runtime": "^7.14.6",
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
     "@madie/madie-auth": "^0.0.1",
```

Comment:
@emotion, @mui/material and axios is already provided as dependency. Do we still need them as devDependencies ?

Reply:
No, not required. Thanks for letting me know I have changed it